content/markdown/01-basic-r/01-01-coding-basics.rmd

```diff
@@ -0,0 +1,118 @@
+
+## R as a Calculator
+
+At the most basic level, we can use R as a (very complicated) calculator. We can perform all the most basic operations that we might be interested in. 
+
+The classic operations (addition, subtraction, multiplication, division) work much as you might expect 
+
+```{r}
+2 + 2
+3 - 2
+3 * 2 # multiplication is denoted by an asterisk
+6 / 2
+2 ^ 3 # exponentiation is denoted by this carrot
+```
+
+You can also ask R to help with more complicated operations such as:
+
+**Modulo operations** (getting the remainder of a division)
+```{r}
+5 %% 2
+```
+
+**Integer Division** (only returns the integer value and not the decimal places)
+
+```{r}
+5 %/% 2
+```
+
+## Boolean Operations
+
+We can ask R to make comparisons between values and give us a boolean (true or false) response for a wide variety of operations. 
+
+We can ask R if two values are equal `==`, inequal `!=`, greater than `>`, less than `<`, greater than *or equal to* `>=`, or less than *or equal to* `<=`. Some examples:
+
+```
+
+```
+```{r}
+2 == 2
+2 != 3
+2 > 4
+```
+
+We can also ask R if two things are true at the same time using the ampersand `&` operator, or if *at least* one statement is true using the vertical bar `|` operator:
+```{r}
+2 == 2 & 3 == 3
+2 == 2 & 3 == 4
+2 == 2 | 3 == 4
+2 == 3 | 3 == 4
+```
+
+We can also chain these operations together, but the order in which we write this out matters. Computers follow order of operations rules just like in math, so we need to think back to BEDMAS (Brackets, Exponents, Division, Multiplication, Addition, Subtraction) from grade school math class! 
+
+**Exercise** Mentally work through what answer (`TRUE` or `FALSE`) R will output when you run this code, then check the answer to see if you were right!
+```{r}
+(2 == 2) & (2 == 3 | 3 == 3)
+```
+
+## Assignment
+
+You'll notice that up to now, we have simply been using numbers, but we can also use variables in R to store the values of numbers. If we wanted a variable `x` with value `2`, we could assign this value in R using the *assignment* operator, `<-`. Once a variable is assigned, that variable will be stored in the environment, and can be called at any later point in the session. 
+
+```{r}
+x <- 2
+x
+```
+
+**Two important notes with assignment:**
+
+1) In R, assignment can be done either through `<-` or `=`. Both will work identically, but the `<-` assignment operator is the more commonly used one and is technically best practice
```

Comment:
This might come much later (and maybe I just haven't gotten to it yet), but one thing I liked about the draft manual that Leila K made was she had a section on writing clean code (where to put spaces, etc.). You are doing it automatically here, of course, as you already know it all, but should we be talking about where and when to put spaces, do we leave this to personal preference? I don't have an answer here, just something to think about.